pages/index.js

```diff
@@ -1,5 +1,6 @@
 import Head from 'next/head';
 import About from '../components/about';
+import { AudioPlayer } from "../components/AudioPlayer"
```

Comment:
```suggestion
import  AudioPlayer  from "../components/AudioPlayer"
```
If we say `export default` we can import `AudioPlayer` without curly brackets.